src/coreclr/jit/scopeinfo.cpp

```diff
@@ -1735,12 +1733,6 @@ void CodeGen::psiBegProlog()
                         {
                             assert(false && "Invalid eightbyte number.");
                         }
-
-                        regType = compiler->GetEightByteType(structDesc, nCnt);
-#ifdef DEBUG
-                        regType = compiler->mangleVarArgsType(regType);
-                        assert(genMapRegNumToRegArgNum((nCnt == 0 ? regNum : otherRegNum), regType) != (unsigned)-1);
-#endif // DEBUG
```

Comment:
I have removed this assert in a couple of places because I don't think it is asserting anything meaningful. The existence and mapping of `genMapRegNumToRegArgNum` is really an internal detail of how `genFnPrologCalleeRegArgs` works, and I expect to be able to remove it once that function is rewritten.